gpii/node_modules/flowManager/test/configs/gpii.flowManager.tests.capture.fakeData.config.json5

```diff
@@ -22,6 +22,13 @@
                 },
                 "target": "{that deviceReporter platformReporter}.options.invokers.reportPlatform",
```

Comment:
Don't forget to add in "gpii.test.integration.mockHolder.windows" to the grades to ensure no further tests pick up environmental dependencies

Reply:
I'm not sure I actually want what's supplied in the windows mocks for these tests after looking through them.  At the moment i have the configs set up the same as the rest of the flowmanager tests which is convenient, and am already simulating the device reporter, solutions registry and resolvers in this config.  The tests are actually simulated on a platform os of `darwin` although I guess I could change them all to `win32`. However, I do actually want to use the regular cross platform json settings handlers because I've provided test files for them, rather than a mocked settings handler. The only remaining thing it looks like the mocks provide are for action handlers, but none of the capture methods use the actions from the lifecycle handler.